packages/vm/src/wasm_backend/engine.rs

```diff
@@ -86,7 +86,7 @@ mod tests {
     #[test]
     fn limit_to_pages_works() {
         // rounds down
-        assert_eq!(limit_to_pages(Size(0)), Pages(0));
+        assert_eq!(limit_to_pages(Size::kilo(0)), Pages(0));
         assert_eq!(limit_to_pages(Size(1)), Pages(0));
```

Comment:
This looks inconsistent now. Should we just add a `::new` function that works like kilo but without a factor?

Reply:
Yes, I think a new function is probably the best option here